packages/ecs-agent/ecs.service

```diff
@@ -13,17 +13,29 @@ RestartSec=1s
 EnvironmentFile=-/etc/ecs/ecs.config
 EnvironmentFile=/etc/network/proxy.env
 Environment=ECS_CHECKPOINT=true
+# Grant ECS tasks access to IMDS
```

Comment:
It's not really IMDS ("instance metadata service"), it's the ECS task metadata endpoint.

Reply:
I see, thanks! I'll use that name.